packages/login/README.md

```diff
@@ -27,7 +27,8 @@ React-registry name: `login`
 |------------------------|:---------:|-------------
 | `showTitle`            |           | Display a title above the dialog.
 | `locale`               |           | ISO Language Code
-
+| `passwordRequest`      |           | Option to open login app in reset mode with pre-filled username. 
```

Comment:
Make clear hear that it is a boolean. "Boolean to open  ...". Username part you can leave because this has nothing to to with the passwordRequest flag.